### PR TITLE
adding-write-velocity-or-displacement-history-option

### PR DESCRIPTION
**Pull request by odakan**
@volkanozsarac

Modify the write method of EzGM.selection:

1) add rtype option
2) write documentation for rtype
3) implement get_history() local method
4) integrate acc. series dir. 1 and 2 just after scaling and before writing to txt

**_get_history(series, dt, type)_:** Cumulatively integrates "series" over [0, t] with "dt" to obtain "type".

**Comparison between EzGM and SeismoSignal:**

EzGM vs SeismoSignal

### CODE REVIEW

File: EzGM/selection.py [hunk 1]
```diff
@@ -281,6 +281,36 @@ def write(self, obj=0, recs=1, rtype='acc', recs_f=''):
         -------
         None.
         """
+        def rename_nga(filename, hist):
```
Comment (volkanozsarac) [inline EzGM/selection.py:284]: Let's make the code more elegant. Nowadays, I try to make the codes as short as possible. I do not think we need these rename functions.

File: EzGM/selection.py [hunk 2]
```diff
@@ -281,6 +281,36 @@ def write(self, obj=0, recs=1, rtype='acc', recs_f=''):
...
+
+            elif hist == 'disp':
+                new_name = filename.replace('ACC', 'DISP')
+
+            else:
+                new_name = filename
+
+            return new_name
+
         def get_history(acc_history, dt, hist):
```
Comment (volkanozsarac) [inline EzGM/selection.py:314]: This function is fine, but lets also do the scaling inside

**Review by volkanozsarac — COMMENTED** [review]
Could you check these changes @odakan?
I do not think we need those rename functions. Let's make the code as short as possible, likewise as elegant as possible.

**Comment by odakan** [comment]
@volkanozsarac, it looks great! 

Everything seems to work fine. Also, I think it is a good idea to save scaling factors as .txt. 

Kudos!
Deniz

**Pull request merged**